packages/nodes-base/nodes/Note/Note.node.json

```diff
@@ -0,0 +1,24 @@
+{
+	"node": "n8n-nodes-base.note",
+	"nodeVersion": "1.0",
+	"codexVersion": "1.0",
+	"categories": [
+		"Core Nodes"
+	],
+	"resources": {
+		"primaryDocumentation": [
+			{
+				"url": "https://docs.n8n.io/nodes/n8n-nodes-base.note/"
+			}
+		]
+	},
+	"alias": [
+		"Comments",
+		"Sticky"
+	],
```

Comment:
did you check with Max on these? this affects search. we should probably add `notes`  as well